ui/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -119,7 +119,12 @@ function handleTelemetry(context: IActionContext, callbackId: string, start: num
         const end: number = Date.now();
         context.telemetry.measurements.duration = (end - start) / 1000;
 
+        const errorPropRegExp: RegExp = /(error|stack|exception)/i;
         // Note: The id of the extension is automatically prepended to the given callbackId (e.g. "vscode-cosmosdb/")
-        ext.reporter.sendTelemetryEvent(callbackId, context.telemetry.properties, context.telemetry.measurements);
+        if (Object.entries(context.telemetry.properties).some(([key, value]) => errorPropRegExp.test(key) && value)) {
+            ext.reporter.sendTelemetryErrorEvent(callbackId, context.telemetry.properties, context.telemetry.measurements);
```

Comment:
Might make sense to send two events in this case so we get *some* telemetry in VSO when errors occur, but could be changed later.